src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderMap.cs

```diff
@@ -47,59 +75,152 @@ private CoseHeaderMap(bool isReadOnly)
             }
         }
 
+        /// <summary>
+        /// Gets the CBOR encoded value associated with the specified label, as a signed integer.
```

Comment:
To reduce confusion later in GetValueAsBytes(label, destination), I'd just remove "CBOR encoded" from all of these, since that's not relevant to a caller.

```suggestion
        /// Gets the value associated with the specified label, as a signed integer.
```